docs/source/changelog.rst

```diff
@@ -11,6 +11,39 @@ Released on December 15, 2023
 Highlights
 ^^^^^^^^^^
 
+Logical Query Planning now available for Dask DataFrames
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Dask DataFrames now make a logical query planning layer available that makes queries
+more efficient. This feature is currently optionally available and requires an explicit opt
+in by users. Set
+
+```
+dask.config.set({"dataframe.query-planning": True})
+```
+
+to activate it. Additionally, you have to install ``dask-expr`` with
+
+```
+pip install dask-expr
+```
+
+to get access to the implementation. This feature is still under active development
```

Comment:
Not a bid deal, but I'd switch the ordering here to say "We've seen promising performance improvements... The features is still under active development..."

Reply:
sure